From ea6f0832bf3879bbc68089399bbb05d411c766d2 Mon Sep 17 00:00:00 2001
From: Marcin Deranek <marcin.deranek@booking.com>
Date: Tue, 13 Dec 2016 12:40:01 +0100
Subject: [PATCH 08/19] DOC: fix small typo in fe_id (backend instead of
 frontend)

Needs to be backported to 1.7 and 1.6 at least.
(cherry picked from commit 6e413ed1ed33e65740bfa1f209383252eb472256)
---
 doc/configuration.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/configuration.txt b/doc/configuration.txt
index 6795166..6596578 100644
--- a/doc/configuration.txt
+++ b/doc/configuration.txt
@@ -13182,7 +13182,7 @@ fc_reordering(<unit>) : integer
 
 fe_id : integer
   Returns an integer containing the current frontend's id. It can be used in
-  backends to check from which backend it was called, or to stick all users
+  backends to check from which frontend it was called, or to stick all users
   coming via a same frontend to the same server.
 
 fe_name : string
-- 
2.10.2

